packages/contract-wrappers/src/contract_wrappers/erc20_proxy_wrapper.ts

```diff
@@ -1,38 +1,39 @@
+import { ERC20ProxyContract } from '@0xproject/abi-gen-wrappers';
+import { ERC20Proxy } from '@0xproject/contract-artifacts';
 import { AssetProxyId } from '@0xproject/types';
 import { Web3Wrapper } from '@0xproject/web3-wrapper';
 import { ContractAbi } from 'ethereum-types';
 import * as _ from 'lodash';
 
-import { artifacts } from '../artifacts';
 import { assert } from '../utils/assert';
 
 import { ContractWrapper } from './contract_wrapper';
-import { ERC20ProxyContract } from './generated/erc20_proxy';
 
 /**
  * This class includes the functionality related to interacting with the ERC20Proxy contract.
  */
 export class ERC20ProxyWrapper extends ContractWrapper {
-    public abi: ContractAbi = artifacts.ERC20Proxy.compilerOutput.abi;
+    public abi: ContractAbi = ERC20Proxy.compilerOutput.abi;
+    public address: string;
     private _erc20ProxyContractIfExists?: ERC20ProxyContract;
-    private _contractAddressIfExists?: string;
     /**
      * Instantiate ERC20ProxyWrapper
      * @param web3Wrapper Web3Wrapper instance to use
      * @param networkId Desired networkId
-     * @param contractAddressIfExists The contract address to use. This is usually pulled from
-     * the artifacts but needs to be specified when using with your own custom testnet.
+     * @param address (Optional) The address of the ERC20Proxy contract. If
```

Comment:
Let's omit `(Optional)` since the `?` in TS denotes that it is optional. Worried we'd update one and not the other in the future.